src/test/unit/helpers/controller/ClaimantContactDetailsHelper.test.ts

```diff
@@ -0,0 +1,70 @@
+import { CaseWithId, YesOrNo } from '../../../../main/definitions/case';
+import { AnyRecord } from '../../../../main/definitions/util-types';
+import { getClaimantContactDetails } from '../../../../main/helpers/controller/ClaimantContactDetailsHelper';
+import claimantContactDetailsJson from '../../../../main/resources/locales/en/translation/claimant-contact-details.json';
+import { mockRequestWithTranslation } from '../../mocks/mockRequest';
+
+describe('Claimant Contact Details Helper', () => {
+  describe('getClaimantContactDetails', () => {
+    const translations: AnyRecord = {
+      ...claimantContactDetailsJson,
+    };
+
+    it('should return claimant details when user is not represented', () => {
+      const userCase: CaseWithId = {
+        firstName: 'John',
+        lastName: 'Doe',
+        address1: '123 Street',
+        address2: 'Apt 1',
+        addressTown: 'London',
+        addressPostcode: 'W1A 1AA',
+        addressCountry: 'UK',
+        email: 'john@example.com',
+      } as CaseWithId;
+
+      const req = mockRequestWithTranslation({ session: { userCase } }, translations);
+
+      const result = getClaimantContactDetails(req);
+      expect(result).toHaveLength(3);
+      expect(result[0].key.text).toBe('Name');
+      expect(result[0].value.text).toBe('John Doe');
+      expect(result[1].key.text).toBe('Address');
+      expect(result[1].value.text).toBe('123 Street, Apt 1, London, W1A 1AA, UK');
+      expect(result[2].key.text).toBe('Email');
+      expect(result[2].value.text).toBe('john@example.com');
+    });
+
+    it('should return legal rep details when user is represented with MyHMCTS', () => {
```

Comment:
Minor point but I guess this test scenario name should be updated to reflect that the rep is not MyHMCTS
```suggestion
    it('should return legal rep details when user is represented', () => {
```